src/app/utils/utils.test.js

```diff
@@ -0,0 +1,19 @@
+import { pluralize } from "./utils";
+
+describe("Util", () => {
+  it("pluralize should correctly pluralize a single item", () => {
+    const singleItems = 1;
+    const label = pluralize(singleItems, "item");
+    expect(label).toBe("item");
+  });
+  it("pluralize should correctly pluralize a multiple item", () => {
+    const multipleItems = 2;
+    const label = pluralize(multipleItems, "item");
+    expect(label).toBe("items");
+  });
+  it("pluralize special cases correctly", () => {
```

Comment:
This test is quite fragile as the list of special cases is internal to the function and cannot be easily tested. Maybe consider moving the special list out of the `pluralize` function and exporting that specials list. You can then loop through that list in the test to assert that it works as expected.